Add powerbiembedded management client with tests, recordings, license, and readme.

mattmazzola opened the pull request:
Added powerBIEmbeddedManagementClient with tests, test traffic recordings, license and reame.

Note: The update workspace collection and delete workspace collection tests are currently excluded since there are bugs with the actual powerbi service.

azurecla commented:
Hi __@mattmazzola__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mattm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

amarzavery commented on test/services/powerbiembedded/powerBIEmbeddedManagementClient-test.js:
```diff
@@ -0,0 +1,273 @@
+    /**
+     * Skipped because there is an error with long-polling operation returning the incorrect location header to cluster url instead of relative url to ARM
+     */
+    it('delete workspace collection', null, function (done) {
```
it.skip should do it.

mattmazzola commented:
Just to clarify, I had originally made the pull request containing test recordings which included authentication information, I removed them from the commit, recorded new tests, and added them in new commit.
